node/src/service/service_parachain.rs

```diff
@@ -231,6 +236,32 @@ where
     })
 }
 
+// TODO(#1040) `build_relay_chain_interface` will be included in polkadot-v0.9.36 in cumulus-client-service here: https://github.com/paritytech/cumulus/commit/babf73bbc6ade358db105580b68aacb91550faa5#diff-52df5aabcc0b97cfe43ce4b88b9dcf52de661934e481f6ae935435f4941b1639R224-R252
```

Comment:
Wasn't this taken from PureStake, as well?

Reply:
No, I could find it in the cumulus repository at a later version.

PureStake just gave me a hint that cumulus already implemented it.

Reply:
I mean, isn't this just this: https://github.com/PureStake/moonbeam/blob/84c39610fcefbbfc57054ccbb8f5a10e9c2cb63d/node/service/src/lib.rs#L501-L528 with the `let` statement replaced with a `match` expression? Not sure if we need to add copyright here.

Reply:
Yes it is the same, but also from [here](https://github.com/paritytech/cumulus/commit/babf73bbc6ade358db105580b68aacb91550faa5#diff-52df5aabcc0b97cfe43ce4b88b9dcf52de661934e481f6ae935435f4941b1639R224-R252).